openlibrary/macros/EditButtons.html

```diff
@@ -10,7 +10,7 @@
 </div>
 
 <div class="cclicense">
-    $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="This link to Creative Commons will open in a new window">CC0</a>. Yippee!')
+    $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="$_('This link to Creative Commons will open in a new window')">$_("CC0")</a>. Yippee!')
```

Comment:
Has this been tested? Nesting translation strings?

Reply:
This nesting looks wrong, and I'd suggest removing the 'Yippee!' rather than translating it :)

Reply:
Okay sure thanks @mekarpeles, @hornc

Reply:
This was marked as resolved, but I don't see any commits implementing the request change.

One way to address this would be to translate the title string separately and pass it in as a parameter. Something like:

```
$title = _('This link to Creative Commons will open in a new window')
$:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="$title">$_("CC0")</a>. Yippee!')

```